wrappers/unity/Assets/RealSenseSDK2.0/Scripts/ProcessingBlocks/RsDepthPrecision.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Intel.RealSense;
+using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using UnityEngine;
+
+public class RsDepthPrecision : RsProcessingBlock
```

Comment:
Given this is a somewhat proprietary thing, maybe we should offer some documentation about what it is trying to show?